config/300-configmappropagation.yaml

```diff
@@ -0,0 +1 @@
+core/resources/configmappropagation.yaml
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
core/resources/configmappropagation.yaml

```

Reply:
Does symbolic link also need trailing newline?

Reply:
No, don't edit this file. @mattmoor your bot should probably ignore symlinks